src/otx/algo/anomaly/padim.py

```diff
@@ -132,3 +131,16 @@ def predict_step(
         if not isinstance(inputs, dict):
             inputs = self._customize_inputs(inputs)
         return AnomalibPadim.predict_step(self, inputs, batch_idx, **kwargs)  # type: ignore[misc]
+
+    def forward(
```

Comment:
If this function is neccesary for both padim and stfpm, how about moving it to `OTXAnomaly` class?

Reply:
The problem is that it calls `self.validation_step` which ends up calling OTXModel's validation step. We need AnomalibModule's validation step